README.md

```diff
@@ -60,16 +60,31 @@ The project's structure is a reflection of the following clean architecture prin
 
 The API specification is compatible with the standard DAS specification here https://github.com/metaplex-foundation/api-specifications
 
+### Contributing
```

Comment:
please move this under developing and running as a subsection. As this applies to both internal and external PRs